backend/spec/views/spree/admin/shared/navigation_footer_spec.rb

```diff
@@ -6,16 +6,39 @@
   let(:user) { FactoryBot.build_stubbed(:admin_user) }
   before do
     allow(view).to receive(:try_spree_current_user).and_return(user)
+    @ability = Object.new.extend(CanCan::Ability)
+    allow(@controller).to receive(:current_ability).and_return(@ability)
```

Comment:
I think `@controller` [can be called as `controller` without the `@` here](https://relishapp.com/rspec/rspec-rails/v/3-7/docs/view-specs/view-spec).